source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/source/mobile/v31/GML2OffRoad.java

```diff
@@ -94,8 +97,14 @@ private OffRoadMobileSource convert(final IsGmlCustomOffRoadMobileSource customM
     final CustomOffRoadMobileSource customVehicleEmissionValues = new CustomOffRoadMobileSource();
     customVehicleEmissionValues.setDescription(customMobileSource.getDescription());
 
-    customVehicleEmissionValues.setCharacteristics(gml2SourceCharacteristics.fromGML(customMobileSource.getCharacteristics(),
-        null, null));
+    final SourceCharacteristics characteristics = gml2SourceCharacteristics.fromGML(
+        customMobileSource.getCharacteristics(), null, null);
+    if (characteristics instanceof OPSSourceCharacteristics opsCharacteristics) {
+      opsCharacteristics.setDiurnalVariation(DiurnalVariation.INDUSTRIAL_ACTIVITY);
+      customVehicleEmissionValues.setCharacteristics(opsCharacteristics);
+    } else {
+      customVehicleEmissionValues.setCharacteristics(characteristics);
+    }
```

Comment:
```suggestion
    final SourceCharacteristics characteristics = gml2SourceCharacteristics.fromGML(
        customMobileSource.getCharacteristics(), null, null);
    if (characteristics instanceof final OPSSourceCharacteristics opsCharacteristics) {
      opsCharacteristics.setDiurnalVariation(DiurnalVariation.INDUSTRIAL_ACTIVITY);
      customVehicleEmissionValues.setCharacteristics(opsCharacteristics);
    }
    customVehicleEmissionValues.setCharacteristics(characteristics);
```
Think this works as well, but might want to check it locally